love/src/components/Facility/Map/Map.jsx

```diff
@@ -19,7 +19,7 @@ this program. If not, see <http://www.gnu.org/licenses/>.
 
 import React, { Component } from 'react';
 import styles from './Map.module.css';
-import Badge from '../../GeneralPurpose/Badge/Badge';
+import { isArray } from 'lodash';
```

Comment:
I think this is not being used.

Reply:
you are correct, applying changes and merging